jetty-util/src/test/java/org/eclipse/jetty/util/PoolTest.java

```diff
@@ -53,7 +52,24 @@
 
     interface Factory
     {
-        Pool<String> getPool(int maxSize);
+        Pool<CloseableHolder> getPool(int maxSize);
```

Comment:
Changes to this class are unrelated with the PR, right?

Reply:
Correct. I wanted to add a few more checks to assert that connections are closed when they should be while working on this PR.